ghp_import.py

```diff
@@ -74,7 +74,8 @@ def try_rebase(remote, branch):
     (rev, ignore) = p.communicate()
     if p.wait() != 0:
         return True
-    cmd = ['git', 'update-ref', 'refs/heads/%s' % branch, rev.strip()]
+    rev = rev.strip().decode('ascii')
```

Comment:
True, but `dec` also does a type check and avoids a call to `decode` is it is not needed, possibly avoiding an error in some situations. Interestingly, the proposed fix in #42 also does the type check, but doesn't decode to ASCII. Seems like my proposed fix or #42 with ASCII would be a better fix.